### PR TITLE
[ADD] aoi_odoo_xmlrpc: AFC-1406 Added module to sync data.

### PR DESCRIPTION
**Pull request by rupareliyahemangi145**

### CODE REVIEW

File: api_odoo_xmlrpc/wizard/api_odoo_synchro.py [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+
+_logger = logging.getLogger(__name__)
+
+
+class RPCProxyOne(object):
+
+    def __init__(self, server, ressource):
+        """Class to store one RPC proxy server."""
+        self.server = server
+        print ("Server URL =-=-", server.server_url)
```
Comment (daerio) [inline api_odoo_xmlrpc/wizard/api_odoo_synchro.py:19]: Please remove calls to print() throughout. Use _logger instead if it is something that is valuable to log, but if it's only for development/testing purposes please remove it.

**Review by daerio — CHANGES_REQUESTED** [review]
Please remove unnecessary logs. I.e calls to print(), maybe _logger as well. If it is valuable information for finding errors keep them as _logger calls.

**Comment by rupareliyahemangi145** [comment]
Ok @daerio I will remove.

**Comment by FredrikArvas** [comment]
@rupareliyahemangi145 
I just spoke to @daerio  and we decided to let you keep the log-lines until we have come further in the development. 
(Daniel did not know that this module will be enhanced during the next few days.)

My suggestion: Keep the log-entries until we are "ready for final testing".

**Comment by rupareliyahemangi145** [comment]
Ok @FredrikArvas

**Pull request merged**